src/components/FilterPanel/FilterPanel.stories.js

```diff
@@ -162,7 +162,8 @@ storiesOf(patterns('FilterPanel'), module)
       </FilterPanelAccordion>
       <FilterPanelAccordion title="Filter accordion 2" count={100}>
         <FilterPanelAccordionItem
-          title="Filter accordion item 1"
+          title={<span>Filter accordion item 1</span>}
+          hoverTitle="Filter accordion item 1"
```

Comment:
This was my fault for not having thought this through when I created these components. Especially because `title` means something in HTML 🤦. 

I'm wondering if a `heading` prop could be introduced instead to replace the current `title` prop and keep the `title` prop to mean the same as it means in HTML.

A transition strategy could be that title is applied when the `heading` is not provided:
```jsx
// Maintain current behavior for users (maybe add a warning when heading does 
// not exist to get people to transition)
<FilterPanelAccordion title="I appear as both the heading and title" />

```

```jsx
// Provide a separate heading and title (similar to title & hoverTitle, but 
// using title to match the HTML attribute name)
<FilterPanelAccordion
  heading={<span>Visible heading</span>}
  title="Hover title that is different"
/>
```

Reply:
@diego-codes that's a great suggestion, but the primary reason I took this path is to avoid a breaking change for existing users.

But I suppose your team is the primary user in this case? So if you are okay with that change & updating your team's usage, this would work for me 👍

Reply:
I'd be fine with the change, especially if you fall back to `title` when a `heading` isn't provided then the change wouldn't be breaking.

Reply:
I mean: it would be "breaking" in that if you are currently passing in a `node` type to `title`, you will start to get proptype errors because the prop has become more restrictive.

Reply:
Oh yeah, that's a good point. 🤔

Reply:
Just realized this is a `feat` so it will be a non-trivial version bump 👍 I think it'll be fine